.github/workflows/tests.yml

```diff
@@ -45,14 +47,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ['7.4', '8.0', '8.1']
+        php-versions: ['7.4', '8.0', '8.1', '8.2']
     name: integration-tests (PHP ${{ matrix.php-versions }})
     steps:
     - uses: actions/checkout@v1
     - name: Install PHP
       uses: shivammathur/setup-php@v2
       with:
         php-version: ${{ matrix.php-versions }}
+        coverage: none
```

Comment:
Later we could introduce coverage if needed, but currently also disabled to avoid installing xdebug